contracts/OwnerUpgradeableProxy.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.4.24;
+
+import "./UpgradeableProxy.sol";
+
+/*
+ * OwnerUpgradeableProxy is an upgradeable proxy that only allows the contract
+ * owner to upgrade and pause the proxy.
+ */
+contract OwnerUpgradeableProxy is UpgradeableProxy {
+  bytes32 private constant ownerPosition = keccak256("gochain.proxy.owner");
```

Comment:
Is this just intended to choose a unique storage location not likely to collide with anything that the user's contract uses?
No need to allow them to override `"gochain.proxy.owner"` with a custom string?

Reply:
Yes, it's meant to be a random address. Subclasses don't need to override it. It's just a memory address that users should avoid but the address space is enormous so that shouldn't be an issue.